synapse_data_warehouse/synapse_aggregate/V2.47.0__create_synapse_aggregate_schema.sql

```diff
@@ -0,0 +1,3 @@
+USE DATABASE {{ database_name }}; --noqa: JJ01,PRS,TMP
+
+CREATE OR REPLACE SCHEMA SYNAPSE_AGGREGATE;
```

Comment:
We need to grant OWNERSHIP of this schema to `synapse_aggregate_all_admin`

Also do we need to add all these patchwork grants for DE? Namely create table/modify/usage

<img width="745" alt="image" src="https://github.com/user-attachments/assets/c8583066-76bb-4baa-9db5-066cf2bb827c" />

Reply:
Could you remind me why we granted these schema privileges to DATA_ENGINEER in the dev DBs?